Modules/Migration.lua

```diff
@@ -118,6 +118,22 @@ local migrationFunctions = {
                 Questie.db.global.isleOfQuelDanasPhase = 1
             end
         end
+    end,
+    [17] = function()
+        -- Preserve previous dungeon hide preference for both new flags
+        local previousHideInDungeons = Questie.db.profile.hideTrackerInDungeons
+
+        Questie.db.profile.minimizeTrackerInCombat = false
+        Questie.db.profile.minimizeTrackerInDungeons = previousHideInDungeons
+        Questie.db.profile.hideTrackerInCombat = false
+        Questie.db.profile.hideTrackerInDungeons = previousHideInDungeons
```

Comment:
We can't set both `Questie.db.profile.minimizeTrackerInDungeons` and `Questie.db.profile.hideTrackerInDungeons` to the same value if `previousHideInDungeons` is `true`.

Imo we should set `Questie.db.profile.hideTrackerInDungeons` to `false`.